prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -137,6 +139,8 @@ if [[ $SYNCMODE == 1 ]]; then
   has_processing_step TPC_DEDX && GPU_CONFIG_KEY+="GPU_global.rundEdx=1;"
   has_detector ITS && TRD_FILTER_CONFIG+=" --filter-trigrec"
 else
+  has_detectors_gpu TPC ITS && ITS_CONFIG_KEY+="ITSCATrackerParam.trackingMode=1;"
+  # echo $ITS_CONFIG_KEY && exit 1
```

Comment:
Please remove debug code, and in case you want to add comments, please put them at the end of an existing line to keep this file as short as possible.